### ISSUE TITLE
If there is something wrong when "bias_trem = false"

### ISSUE THREAD

**Issue by lawrencewxj** [kai-xie/Neural-Network-Model-Compressing#16]
Dear Kai, thank you for your work. 
At dns_conv_layer.cpp's line 74-76, when forward In line 75, There is no bias_trem judgment here. If bias_term= false, then, there is no this->blobs_[2], so it maybe go wrong. The same with backward and dns_conv_layer.cu. I do not know if my findings are correct. If it's right, maybe we can add a if to correct it. @kai-xie 
```
const Dtype* weight = this->blobs_[0]->cpu_data();    //line 74
Dtype* weightMask = this->blobs_[2]->mutable_cpu_data(); // line 75
Dtype* weightTmp = this->weight_tmp_.mutable_cpu_data(); //line 76
```

**Comment by kai-xie** [comment]
@lawrencewxj you are right. This is a bug. Just fixed it. Let me know if there are other bugs.
